playground/AspireEventHub/EventHubs.AppHost/ehstorage.module.bicep

```diff
@@ -10,8 +10,8 @@ param principalId string
 param principalType string
 
 
-resource storageAccount_X1L2R0Ykm 'Microsoft.Storage/storageAccounts@2022-09-01' = {
-  name: toLower(take(concat('ehstorage', uniqueString(resourceGroup().id)), 24))
+resource storageAccount_59pNEh4K4 'Microsoft.Storage/storageAccounts@2022-09-01' = {
```

Comment:
Do these names change _every time_ the bicep is generated? If so, how do we expect people to do something like infra synth and easily diff when there are so many diffs.

Reply:
We changed the formula we use for resource name generation deliberately to address some linter warnings. This bicep was first generated using the old method and had not been updated until I regenerated everything.

These should be stable moving forward. It's a good call out though as we cannot change these moving forward.